pkg/controllers/policycache/controller.go

```diff
@@ -95,7 +105,8 @@ func (c *controller) reconcile(ctx context.Context, logger logr.Logger, key, nam
 		return err
 	}
 	// TODO: check resource version ?
-	c.cache.Set(key, policy)
+	subresourceGVKToKind := getSubresourceGVKToKindMap(policy, c.client)
```

Comment:
Remove TODO?

Reply:
Not yet, I think this TODO is for using the resource version as a part of the key (instead of just the kind), so this is still TODO.